pkg/manifests/manifests.go

```diff
@@ -254,6 +257,38 @@ func DaemonSet(component string) (*appsv1.DaemonSet, error) {
 	return ds, nil
 }
 
+func MachineConfig(component string) (*machineconfigv1.MachineConfig, error) {
+	if err := validateComponent(component); err != nil {
```

Comment:
this is too broad: both the MachineConfig and the SecurityContext make sense only for `component==rte`.
But can be fixed on a later PR

Reply:
do you want to move it under the `rte` package?

Reply:
no need, just drop the call to `validateComponent` and use a plain
```
if component == rte {
...
```